### PR TITLE
Added eu.org for free eu.org domains

### PR DESCRIPTION
**Pull request by anzepintar**
I added link to website from which you can get free eu.org domains

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1383,6 +1383,7 @@ Table of Contents
 ## Other Free Resources
 
   * [education.github.com](https://education.github.com/pack) — Collection of free services for students. Registration required
+  * [eu.org](https://nic.eu.org) — Free eu.org domain. Request is usually approved in 14 days.
```
Comment (FatGrizzly) [inline README.md:1386]: Correction: It's a subdomain, not a domain

**Review by FatGrizzly — CHANGES_REQUESTED** [review]
Correction: It's a subdomain, not a domain

File: README.md (hunk 1 above)
Comment (anzepintar) [reply README.md:1386]: no it is a second level domain like co.uk

Comment (FatGrizzly) [reply README.md:1386]: image

**Comment by ripienaar** [comment]
thanks

**Comment by EDM115** [comment]
@ripienaar EU.ORG no longer attributes subdomains for ~1/2 years, they don't answer to emails and many people are starting to fear for their existing domains if they cease to exist overnight.  
maybe it should be removed or at the very least add a warning about this situation. The "The request is usually approved in 14 days." is no longer a valid claim

**Comment by ripienaar** [comment]
Yup, lets remove it, please send a PR

**Pull request merged**